gvsbuild/projects/glib.py

```diff
@@ -26,13 +26,11 @@ def __init__(self):
         Project.__init__(
             self,
             "glib",
-            archive_url="https://download.gnome.org/sources/glib/2.72/glib-2.72.4.tar.xz",
```

Comment:
Can this be build locally?

latest glib was updated to use pcre2, previously it was downloading pcre when building glib. Now it is needed to add  a pcre2 project, then an explicit dependency from glib to pcre2.